corehq/sql_db/util.py

```diff
@@ -323,6 +323,9 @@ def select_db_for_read(weighted_dbs):
             ]
 
     """
+    if not weighted_dbs:
+        return
+
```

Comment:
possibly a naive question because of no context on this,
in what case would `weighted_dbs` be blank?

Reply:
If there are no standby databases configured for the django alias. See https://github.com/dimagi/commcare-hq/blob/9e5005e03aa25d207753c32dff67cd2c1120c9c4/settings.py#L823-L833 and https://github.com/dimagi/commcare-hq/blob/9e5005e03aa25d207753c32dff67cd2c1120c9c4/settings.py#L781-L791